tests/Kernel/Paragraphs/ContentRowTest.php

```diff
@@ -190,6 +190,23 @@ public function testSideMenuTranslation(): void {
     $left_column = $crawler->filter('.ecl-row .ecl-col-md-3.ecl-u-z-navigation');
     $this->assertContains('French page navigation', $left_column->html());
     $this->assertContains('French rich text title', $left_column->html());
+
+    // Create the main content row paragraph with empty title.
```

Comment:
Its a bit weird to add this test inside the paragraph translation test, but I would like to have the next reviewers input before asking for changes.

Reply:
We are testing if the fallback title gets translated as well.

Reply:
Fair enough :+1:

Reply:
Yes. I agree. It is make sense to move this code in other place.